db/migrate/20200414123737_extract_tasks_source_ref_from_context.rb

```diff
@@ -0,0 +1,21 @@
+class ExtractTasksSourceRefFromContext < ActiveRecord::Migration[5.2]
+  def up
+    Task.find_in_batches.each do |tasks|
```

Comment:
Super minor, but if you're going to do `.find_in_batches { |tasks| tasks.each { |task| } }` this is basically just `.find_each { |task| }`